lib/src/resources/websocket.dart

```diff
@@ -2,11 +2,22 @@ part of 'common.dart';
 
 class Websocket extends Resource {
   late $wp.WebsocketClient _websocketClient;
+  late $wp.WebsocketHandlerClient? _websocketHandlerClient;
+
+  Websocket(String name,
+      {$p.ResourcesClient? client,
+      $wp.WebsocketClient? websocketClient,
+      $wp.WebsocketHandlerClient? websocketHandlerClient})
+      : super(name, client) {
+    if (websocketClient == null) {
+      final channel = createClientChannelFromEnvVar();
```

Comment:
should we be using shared or even a singleton channel here to maximise channel reuse?

Creating dedicated channels could lead to performance problems for larger functions, or when running on smaller infrastructure.

https://grpc.io/docs/guides/performance/